sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/FormRecognizerModelFactory.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.AI.FormRecognizer.Models;
+using Azure.AI.FormRecognizer.Training;
+
+namespace Azure.AI.FormRecognizer
```

Comment:
is there a way to change the namespace? what is the repercussion for this in FR? https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerModelFactory.cs#L11

Also, we are in code freeze for anything that modifies the public API for FR

Reply:
Yes, you can change namespace and type name similar to REST client: 
```cs
    [CodeGenType("GeneratedModelFactory")]
    partial class MyModelFactory
```